Gopkg.lock

```diff
@@ -41,7 +41,7 @@
   branch = "master"
   name = "github.com/giantswarm/gsclientgen"
   packages = ["."]
-  revision = "39bc522c6c6fdd85654480e260a887117b3771ec"
+  revision = "9cc9a2db760c88ec5b26d7c1b10830e1c9b67bd5"
```

Comment:
This here implies the removed kubernetes version property for cluster creation. Note that this property was already not set anymore when creating clusters because we use the release version entirely.